homeassistant/components/group/cover.py

```diff
@@ -200,6 +200,13 @@ def current_cover_tilt_position(self):
         """Return current tilt position for all covers."""
         return self._tilt_position
 
+    @property
+    def state_attributes(self):
+        """Return the state attributes for the cover group."""
+        data = getattr(super(), "state_attributes", {})
+        data[ATTR_ENTITY_ID] = self._entities
+        return data
```

Comment:
```suggestion
    def device_state_attributes(self):
        """Return the state attributes for the cover group."""
        return {ATTR_ENTITY_ID: self._entities}
```

Reply:
I knew there had to be something like that.
Can't believe I missed it. It's [right there in the docs](https://developers.home-assistant.io/docs/core/entity#generic-properties). Thanks!